README.md

```diff
@@ -1,6 +1,197 @@
+- Somewhere we need to discuss our plans for `X-Frame-Options`, CSP `frame-ancestors`, and portal-specific variants of those? This might be the place for that?
+
+### Rendering
+
+TODO:
+
+- Discuss rendering. Full viewport size at all times? Resizing OK or no? I'm sketchy on the plan here.
+- Discuss practices and patterns for authors of portaled sites, e.g. how to adapt on activation.
+- Discuss practices and patterns for authors of portaling sites, e.g. how to create a prerender (with a `display: none` portal) or an animated transition
+- Maybe this is where we discuss fallback content for non-supporting browsers?
+
+### Interactivity
+
+TODO:
+
+- Discuss how portals are only trivially interactive, like buttons or links. Not like iframes.
+- Discuss focus. (Again, like buttons or links.)
+- Did we end up adding a default click behavior like links?
+- Discuss scrolling, including the problem of scroll handoff and its importance. Note that scroll handoff is a ??? in spec terms but we promise to spec something interoperably implementable, somehow.
+
+### Accessbility
+
+TODO:
+
+- Discuss role and description.
+- Discuss best practices for authors of portaling pages.
+- Explicitly mention how authors should respect `prefers-reduced-motion` (and how this might happen automatically if they already turn off all CSS transitions/animations when that media query tests true).
+
+### Session history, navigation, and bfcache
+
+TODO:
+
+- One-sentence reiteration of how activation is a navigation.
+- Talk about how session history is tricky.
+- Talk about how bfcache is tricky.
+- Outline our solution for these, in terms of observable effects for users (not in terms of spec primitives).
+- Maybe compare/contrast with iframes. Is it helpful or confusing to talk about how portals are more like popups than iframes? (I.e. they are auxiliary browsing contexts in spec terms.)
+
+## Summary of differences between portals and iframes
+
+TODO:
+
+- Maybe introduce by telling people to think of portals as super-links, not super-iframes. At least from a user's perspective; from the spec and implementer perspective they're more related to iframes, and for web developers they're kind of in between.
+- Reiterate everything from the above sections in one convenient place.
+- Include minor things as well (like, doesn't appear in `window.frames`).
+
+## Alternatives considered
+
+TODO:
+
+- A new attribute on iframes (text exists in explainer.md but is very implementer-focused)
+- Other (historical?) solutions to prerendering
+- Other (historical?) solutions to navigation transitions
+- Adding activation ("promotion") to iframes (text exists in explainer.md but is very implementer-focused)
+- Using the fullscreen API (text exists in explainer.md but is very implementer-focused)
+- Allowing cross-origin communication and storage
+
+## Acknowledgments
```

Comment:
```suggestion
## Security and privacy considerations

TODO

### Security/Privacy Questionnaire

This section contains answers to the [W3C TAG Security and Privacy Questionnaire](https://w3ctag.github.io/security-questionnaire/).

1. TODO

## Stakeholder Feedback

* W3CTAG: ?
* Browsers:
  * Safari: ?
  * Firefox: ?
  * Samsung: ?
  * UC: ?
  * Opera: ?
  * Edge: ?
* Web developers: ?

## Acknowledgments
```

From https://w3ctag.github.io/explainers#stakeholder-feedback--opposition and general good practice. It's possible the questionnaire should live in a separate document.

Reply:
I think a separate document is probably best for the questionnaire. I'll add a skeleton for it.